agent/bin/kontena-agent

```diff
@@ -31,6 +31,8 @@ Kontena::Logging.initialize_logger(STDOUT, log_level)
 
 agent = Kontena::Agent.new(
   api_uri: api_uri,
-  api_token: api_token
+  api_token: api_token,
+  ssl_verify: ENV['KONTENA_SSL_VERIFY'],
+  ssl_ca: ENV['KONTENA_SSL_CA'],
```

Comment:
Hmm... somehow just replacing `KONTENA_SSL_CA=` with `SSL_CERT_FILE=` didn't work... keeping this for now.